src/github/copilotRemoteAgent.ts

```diff
@@ -601,6 +601,14 @@ export class CopilotRemoteAgentManager extends Disposable {
 			return `${header}\n\n${collapsedContext}`;
 		};
 
+		let isTruncated = false;
+		if (problemContext && (problemContext.length + prompt.length >= MAX_PROBLEM_STATEMENT_LENGTH)) {
```

Comment:
the check is for the sum of context+prompt, but the overflow (availableLength) is only based on prompt length. That seems wrong